xml/article_virtualization.xml

```diff
@@ -273,21 +274,21 @@
     Standard real-time dangers apply in that if the intention is to run a
     compute intensive application with realtime priority, then the user must
     ensure that kernel threads cannnot starve. A simple precaution is to use
-    <option>rtkthread=prio</option> and <option>rtworkqueues=prio</option>
-    kernel boot parameters, with priority set higher than anything that may
+    the <option>rtkthread=prio</option> and <option>rtworkqueues=prio</option>
+    kernel boot parameters, with priorities set higher than anything that may
     dominate a CPU. This is not strictly real-time capable, but it is safer
```

Comment:
"with priorities set higher than anything that may dominate a CPU" confuses me.

Reply:
I have reworded it; please let me know if it is less confusing.

Reply:
Thanks! Rereading it + the rewording helped me understand it. I have added another suggestion that tries to clarify it further~.